addons/l10n_in_ewaybill_stock/models/l10n_in_ewaybill.py

```diff
@@ -154,6 +154,15 @@ def _check_lines(self):
             return error_message
         return super()._check_lines()
 
+    def _check_state(self):
+        error_message = super()._check_state()
+        if not self.picking_id:
+            return error_message
+        picking_state = self.picking_id.state
+        if picking_state != 'done' or (self._is_incoming() and picking_state != 'assigned'):
```

Comment:
if not done is giving an error and then incoming with done is also giving error, it has to be outgoing and done?